graph-db/package.json

```diff
@@ -8,7 +8,8 @@
     "test": "jest --config jest.config.js",
     "test-coverage": "jest --coverage",
     "ci-test": "jest --coverage --config jest.config.junit_output.js",
-    "bootstrap": "serverless invoke local -f bootstrapGremlinServer -e CMR_ROOT=\"http://cmr.uat.earthdata.nasa.gov\" -e PAGE_SIZE=2000 -e GREMLIN_URL=\"ws://localhost:8182/gremlin\""
+    "bootstrap": "serverless invoke local -f bootstrapGremlinServer -e CMR_ROOT=\"http://cmr.uat.earthdata.nasa.gov\" -e PAGE_SIZE=2000 -e GREMLIN_URL=\"ws://localhost:8182/gremlin\" --stage=sit",
```

Comment:
The GREMLIN_URL setting does not match the `stage` sit setting. I think the stage should be local dev here.

Reply:
Not in this case. This is for bootstrapping a local gremlin server, so I renamed the command to `bootstrap-local` to make that more clear

Reply:
So that the serverless config doesn't throw errors when it's trying to fill parameter values